docs/preset-env.md

```diff
@@ -132,9 +132,7 @@ We recognize this isn’t ideal and will be revisiting this in Babel v8.
 
 `boolean`.
 
-You may also target browsers supporting ES Modules (https://www.ecma-international.org/ecma-262/6.0/#sec-modules). When specifying this option, the browsers field will be ignored. You can use this approach in combination with `<script type="module"></script>` to conditionally serve smaller scripts to users (https://jakearchibald.com/2017/es-modules-in-browsers/#nomodule-for-backwards-compatibility).
-
-> _Please note_: when specifying the esmodules target, browsers targets will be ignored.
+You may also target browsers supporting ES Modules (https://www.ecma-international.org/ecma-262/6.0/#sec-modules). When the `esmodules` target is specified, the `browsers` target will be ignored. It _will_, however, intersect with `browserslist`'s targets. You can use this approach in combination with `<script type="module"></script>` to conditionally serve smaller scripts to users (https://jakearchibald.com/2017/es-modules-in-browsers/#nomodule-for-backwards-compatibility).
```

Comment:
When I told you that `esmodules` is intersected I was only considering the top-level option :sweat_smile: 

When using `@babel/preset-env`'s option, both `browsers` and `.browserslistrc` are ignored for backward compat reasons.

Reply:
@nicolo-ribaudo Changed.